packages/app-page-builder/src/admin/views/PageBlocks/BlocksByCategoriesDataList.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { i18n } from "@webiny/app/i18n";
+import { useRouter } from "@webiny/react-router";
+import { useQuery } from "@apollo/react-hooks";
```

Comment:
Can you clarify please, what is this view? Can you provide a screenshot?

Reply:
![image (12)](https://user-images.githubusercontent.com/77202393/176856014-22c004ca-8482-480a-91fc-e848ef0dcbe2.png)
Here is a screenshot

Design link - https://miro.com/app/board/uXjVO1XGqwg=/?share_link_id=951919244316